### PR TITLE
fix #5815 feat(nimbus): add support for focus/klar on android

### LINKED ISSUE #5815: Add support for Focus/Klar on Android
We're [integrating Nimbus](https://github.com/mozilla-mobile/focus-android/pull/4952) into Focus/Klar for Android and I believe one of the next steps is to enable it in experimenter as well. Filing this issue for the work that needs to be done for that. 🙂 

cc: @travis79

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-1438)

### PR DESCRIPTION
**Pull request by jaredlockhart**
Because

* We'd like to support Focus and Klar on android

This commit

* Adds application configs for Focus and Klar
* We need to remember to add the `No Feature` fixtures for every new application
* Added a test to make sure those fixtures are added so we don't forget in the future
* This is only temporary until we move feature configs over to application repos and consume them the same way we do outcomes and take them out of the database altogether

### CODE REVIEW

**Comment by jaredlockhart** [comment]
Blocked on https://github.com/mozilla/experimenter/issues/6081

File: app/experimenter/experiments/constants/nimbus.py [hunk 1]
```diff
@@ -96,11 +130,7 @@ class NimbusTargetingConfig:
     description="All users",
     targeting="",
     desktop_telemetry="",
-    application_choice_names=(
-        Application.DESKTOP.name,
-        Application.FENIX.name,
-        Application.IOS.name,
-    ),
+    application_choice_names=[a.name for a in Application],
```
Comment (lmorchard) [inline app/experimenter/experiments/constants/nimbus.py:133]: :+1:

File: app/experimenter/experiments/tests/test_models/test_models_nimbus.py [hunk 2]
```diff
@@ -1159,3 +1163,19 @@ def test_formats_str_if_no_message_set(self):
...
+class TestNimbusFeatureConfig(TestCase):
+    @parameterized.expand(list(NimbusExperiment.Application))
+    def test_no_feature_fixture_exists(self, application):
+        application_config = NimbusExperiment.APPLICATION_CONFIGS[application]
+        self.assertTrue(
+            NimbusFeatureConfig.objects.filter(
+                name__startswith="No Feature", application=application
+            ).exists(),
+            (
+                f"A 'No Feature {application_config.name}' FeatureConfig fixture "
```
Comment (lmorchard) [inline app/experimenter/experiments/tests/test_models/test_models_nimbus.py:1177]: :+1:

**Review by lmorchard — APPROVED** [review]
LGTM! :+1:

**Pull request merged**